pkg/ansible/controller/reconcile.go

```diff
@@ -249,6 +255,12 @@ func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconc
 	return reconcileResult, nil
 }
 
+func printEventStats(statusEvent eventapi.StatusJobEvent) {
+	fmt.Printf("\n--------------------------- Ansible Task Status Event StdOut  -----------------\n")
+	fmt.Println(statusEvent.StdOut)
+	fmt.Printf("\n-------------------------------------------------------------------------------\n")
+}
+
```

Comment:
Maybe I'm missing something but why do we use `fmt.Printf` here but `logger.` in other places.

Reply:
To print the data formatted such as we do here: https://github.com/operator-framework/operator-sdk/pull/2321

Reply:
Yeah, if you put the ansible stdout through the logger it gets mangled completely since it is JSON serialized